src/sentry/models/rule.py

```diff
@@ -42,6 +55,13 @@ class Rule(Model):
         choices=((RuleStatus.ACTIVE, "Active"), (RuleStatus.INACTIVE, "Inactive")),
         db_index=True,
     )
+    # type is currently used as a way to distinguish rules created specificly
+    # for use in other parts of the product (e.g. monitor alerting rules)
+    type = BoundedPositiveIntegerField(
+        default=RuleType.ISSUE,
+        choices=RuleType.as_choices(),
+        db_index=True,
```

Comment:
I think you'd be better off with an index on `project, type`. We're rarely going to be just filtering on type

Reply:
good point